specification/resources/resource-manager/readme.md

```diff
@@ -81,6 +81,80 @@ tag: package-changes-2022-05
 tag: package-snapshots-2022-11
 ```
 
+### Tag: package-policy-2023-04
+
+These settings apply only when `--tag=package-policy-2023-04` is specified on the command line.
+
+``` yaml $(tag) == 'package-policy-2023-04'
+input-file:
+- Microsoft.Authorization/stable/2020-09-01/dataPolicyManifests.json
+- Microsoft.Authorization/stable/2023-04-01/policyDefinitions.json
+- Microsoft.Authorization/stable/2023-04-01/policyDefinitionVersions.json
+- Microsoft.Authorization/stable/2023-04-01/policySetDefinitions.json
+- Microsoft.Authorization/stable/2023-04-01/policySetDefinitionVersions.json
+- Microsoft.Authorization/stable/2023-04-01/policyAssignments.json
+- Microsoft.Authorization/preview/2022-07-01-preview/policyExemptions.json
+- Microsoft.Authorization/preview/2022-08-01-preview/policyVariables.json
+- Microsoft.Authorization/preview/2022-08-01-preview/policyVariableValues.json
+
+# Needed when there is more than one input file
+override-info:
+  title: PolicyClient
+
+suppressions:
+  - code: PathForPutOperation
+    from: policyDefinitions.json
+    reason: Policy definitions can be created at management group or subscriptions
+  - code: PathForPutOperation
+    from: policySetDefinitions.json
+    reason: Policy sets can be created at management group or subscriptions
+  - code: PathForPutOperation
+    from: policyAssignments.json
+    reason: Policy assignments can be created at management group or subscriptions
+  - code: PathForPutOperation
+    from: policyDefinitionVersions.json
+    reason: Policy definition versions can be created at management group or subscriptions
+  - code: PathForPutOperation
+    from: policySetDefinitionVersions.json
+    reason: Policy set versions can be created at management group or subscriptions
+  - code: DeleteResponseBodyEmpty
+    from: policyAssignments.json
+    reason: Policy assignment body is returned on delete and this must match API
+  - code: RequestSchemaForTrackedResourcesMustHaveTags
+    from: policyAssignments.json
+    reason: Policy assignments are not tracked resources
+  - code: RepeatedPathInfo
+    from: policyAssignments.json
+    reason: Service requires the scope to be in the body
+  - code: RepeatedPathInfo
+    from: policyDefinitionVersions.json
+    reason: Service requires the version to be in the body
+  - code: RepeatedPathInfo
+    from: policySetDefinitionVersions.json
+    reason: Service requires the version to be in the body
+  - code: PutResponseSchemaDescription
+    from: policyAssignments.json
+    reason: Service only returns 201 on all successful PUTs
+  - code: PutResponseSchemaDescription
+    from: policyDefinitions.json
+    reason: Service only returns 201 on all successful PUTs
+  - code: PutResponseSchemaDescription
```

Comment:
>PutResponseSchemaDescription [](http://example.com/codeflow?start=10&length=28)

Please do not suppress errors which can be resolved by updating swagger.
Use this one for example, it is a description error, which can be resolved by updating the `description`.
[https://github.com/Azure/azure-openapi-validator/blob/main/docs/put-response-schema-description.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/put-response-schema-description.md)

Please remove the unnecessary suppression and resolve them by updating the swagger.

Reply:
I can't change the swagger as the service will only return a 201.  Don't know why but it has always been that way and changing that for existing APIs is too risky.  I updated the new APIs to support this.

Reply:
are you sure?  I had this before and got flagged even though the word "create" is in the description.  I mean i can try again if you want.

Reply:
> are you sure? I had this before and got flagged even though the word "create" is in the description. I mean i can try again if you want.

Could you provide the lint-diff commit link that you tried before and still got the error? If it's really the lint-diff's problem, we can suppress this check.

Reply:
here you go: [https://github.com/Azure/azure-rest-api-specs/pull/24542/checks?check_run_id=15374801348](https://github.com/Azure/azure-rest-api-specs/pull/24542/checks?check_run_id=15374801348)

PutResponseSchemaDescription 
Any Put MUST contain 200 and 201 return codes.
Location: Microsoft.Authorization/stable/2023-04-01/policyDefinitions.json#L77

Reply:
> I can't change the swagger as the service will only return a 201. Don't know why but it has always been that way and changing that for existing APIs is too risky. I updated the new APIs to support this.

Since new APIs support this, could you remove all suppressions for `policyDefinitionVersions.json` and `policySetDefinitionVersions.json`, then resolve the errors for these 2 new set of APIs?

Reply:
I checked and those suppressions are there because that is how the service behaves or maintain some uniformity between the existing api's and the new ones.  So RepeatedPathInfo,PathForPutOperation, and ResourceNameRestriction are legitimate

Reply:
Is there any specific reason that the newly added APIs must reuse existing implementations and cannot comply the lint rules?

Reply:
There are 3 suppressions in play here.  Below is the reason for each one (summarized in the .md file)

1) PathForPutOperation - This requires all PUTs to be at the sub/resource group level.  These new policy types, like the previous ones, can be PUT at the management group level.  So this is needed.

2) ResourceNameRestriction - This wants me to add the 'pattern' to a parameter.  However, I am using a common-type that I don't own and don't want to validate the naming conventions.

3) RepeatedPathInfo - This says that there is no need to have the name in the URL path and in the body.  Unfortunately this is the one that we need to be consistent with previous APIs.  These new "version" APIs allow customers to version their policies resource types so they need to be essentially the behavior.  I can see about removing this one

Reply:
Thanks for your comments. We will review it. Please reach out to Azure Breaking Changes Reviewers azbreakchangereview@microsoft.com to get a breaking changes approval.

Reply:
Breaking change approved...  also removed the RepeatedPathInfo suppression.  Can't remove the other ones